test/test-application/src/index.ts

```diff
@@ -386,13 +587,7 @@ async function testWait(client: EchoClient) {
 }
 
 async function main() {
-  const showcaseServer = new ShowcaseServer();
```

Comment:
Just a reminder to have these lines back before you merge it - we still need it to download and run showcase server in the real CI.

Reply:
Ack, we definitely will